tests/contrib/test_torch.py

```diff
@@ -174,6 +177,25 @@ def test_torch_import(tmp_path, net, torchsave):
     assert isinstance(meta.model_type, TorchModel)
 
 
+def test_torch_import_in_separate_shell():
+    tmp_path = mkdtemp()
+    path = make_posix(os.path.join(tmp_path, "model"))
+    m = torch.nn.Linear(5, 1)
+    torch.save(m, path)
+    x = subprocess.run(
+        [
+            "python",
+            "-c",
+            f"from mlem.api import import_object; from mlem.contrib.torch import TorchModelImport; import_object('{path}', type_=TorchModelImport.type)",
```

Comment:
probably easier to use mlem cli?

Reply:
as in? I have made it simpler btw, but still didn't use `mlem cli` stuff

Reply:
I assume Mike meant something like `mlem pprint {path}`, but don't see big problem with this solution)